src/components/projects/View.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import './Projects.scss';
+import CustomTable from '../Table/CustomTable';
+import { tableColumns } from './components/columns';
+import { setPageData } from './modules/actions';
+
+const Projects = ({
+  dataSource,
+  totalCount,
+  isLoading,
+  actions,
+}) => {
+  const [selectedProjectStatus, setSelectedProjectStatus] = useState('All Projects');
+  const [searchInput, setSearchInput] = useState('');
+
+  const handleSearchChange = (input) => {
+    setSearchInput(input);
+  };
+
+  useEffect(() => {
+    fetchData(1, 10);
```

Comment:
Leave this for now. But I think this will become really messy in future with all this numbers. My personal opinion is to reset page number and page size in redux state. But for now leave